static/500.html

```diff
@@ -0,0 +1,207 @@
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <meta charset="utf-8">
+  <title>The Gazelle</title>
+
+  <!-- Mobile Specific Metas
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <meta name="viewport" content="initial-scale=1, maximum-scale=1">
+
+  <!-- FONT
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,700" rel="stylesheet">
+
+  <!-- Style
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <style>
+  .fontErr {
+    text-align: center; 
+    vertical-align: middle;
+    line-height: 0.5rem; 
+
+
+  }
+  .centered {
+    text-align: center; 
+    margin-left: auto; 
+    margin-right: auto; 
+  }
+
+  .container {
+  position: relative;
+  width: 100%;
+  max-width: 960px;
+  margin: 0 auto;
+  padding: 0 10px;
+  box-sizing: border-box; }
+
+   .headerLogo {
+    margin-left: auto; 
+    margin-right: auto;
+    width: 8%;
+    
+  }
+
+  .column,
+  .columns {
+   width: 100%;
+   float: left;
+   box-sizing: border-box; }
+
+
+/* For devices larger than 400px */
+@media (min-width: 400px) {
+  .container {
+    width: 85%;
+    padding: 0; }
+}
+
+/* For devices larger than 550px */
+@media (min-width: 550px) {
+  .container {
+    width: 80%; }
+  .column,
+  .columns {
+    margin-left: 4%; }
+  .column:first-child,
+  .columns:first-child {
+    margin-left: 0; }
+  .header.column, .headerLogo { 
+    width: 5%; margin-left: auto; margin-right: auto; padding-bottom: 0;   
+  }
+
+  }
+
+
+  body {
+  font-size: 1.5em; 
+  font-family: 'Roboto', sans-serif;
+  color: #484848;
+  text-align: center; }
+
+
+/* Typography
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+h1,h6, p {
+  margin-top: 1.5rem;
+  margin-bottom: 0;
+}
+h1 { font-size: 2.5rem; line-height: 1.2;  }
+h6 { font-size: 0.8rem;  padding-bottom: 0;}
+p {font-size: 0.9rem;}
+
+/* Larger than phablet */
+@media (min-width: 550px) {
+  h1 { font-size: 3.5rem; }
+  h6 { font-size: 0.95rem; }
+  p { font-size: 1.1rem; }
+}
+@media (min-width: 700px) {
+  h1 {font-size: 5.0rem;}
+  h6 {font-size: 1.2rem;}
+  p {font-size: 1.3rem;} 
+}
+
+p {
+  font-family: 'Roboto', sans-serif;
+  color: #000000;
+  margin-top: 0; 
+  font-weight: 300; 
+}
+
+
+code {
+  vertical-align: middle; 
+  padding: 0.1rem 0.1rem;
+  margin-left: auto;
+  margin-right:auto;
+  white-space: nowrap;
+  background: #F1F1F1;
+  border: 1px solid #E1E1E1;
+  border-radius: 0px; }
+pre > code {
+  display: block;
+  padding: 0.5rem 0.5rem;
+  white-space: pre; }
+
+span {
+  text-indent: 1.5em; 
+  margin-left: 
+  width: 100%; 
+}
+
+
+
+
+  /* Self Clearing Goodness */
+.container:after,
+.row:after,
+.u-cf {
+  content: "";
+  display: table;
+  clear: both; }
+
+
+
+
+  </style>
+
+
+  <!-- Add Gazelle Favicon 
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link rel="icon" type="image/png" href="./favicons/favicon-16x16.png">
+
+</head>
+<body>
+  <header>
```

Comment:
This `<header>` tag isn't really necessary. Try moving the logo into the same `<div>` as the main content. The top margin could make it seem less pressed up against the top of the page.